packages/boot/src/types.ts

```diff
@@ -125,3 +126,15 @@ export interface Bootable {
    */
   booters(...booterClasses: Constructor<Booter>[]): Binding[];
 }
+
+/**
+ * `@booter` decorator
```

Comment:
Please explain the purpose of this decorator, why should people use it and especially how it affects `@config()` behavior.

Reply:
Hmm, I just realized that api-extractor/api-documenter has trouble handling two artifacts with case-sensitive names, for examle, `Booter` interface and `@booter` decorator`. As a result, the docs for `@booter` is overridden by `Booter`. Let me open an issue against their repo and see how it goes.

Reply:
See https://github.com/microsoft/web-build-tools/issues/1281